jax/_src/numpy/lax_numpy.py

```diff
@@ -6406,13 +6406,29 @@ def _quantile(a, q, axis, interpolation, keepdims, squash_nans):
     raise ValueError("interpolation can only be 'linear', 'lower', 'higher', "
                      "'midpoint', or 'nearest'")
   a, q = _promote_dtypes_inexact(a, q)
+  keepdim = []
   if issubdtype(a.dtype, np.complexfloating):
     raise ValueError("quantile does not support complex input, as the operation is poorly defined.")
   if axis is None:
     a = ravel(a)
     axis = 0
   elif isinstance(axis, tuple):
-    raise NotImplementedError("Tuple values for axis are not implemented")
+    keepdim = list(shape(a))
+    nd = ndim(a)
+    axis = tuple([_canonicalize_axis(ax, nd) for ax in axis])
+
+    for ax in axis:
+      keepdim[ax] = 1
+
+    keep = set(range(nd)) - set(axis)
+    nkeep = len(keep)
+    # swap axis that should not be reduced to front
+    for i, s in enumerate(sorted(keep)):
+      a = a.swapaxes(i, s)
+    # merge reduced axis
+    a = reshape(a, shape(a)[:nkeep] + (-1,))
```

Comment:
swapping axes and reshaping can be somewhat expensive. I think `lax.reshape` is general enough to do this axis swapping plus the reshape in a single XLA call: we should use that here.

Reply:
Totally, did not know about the existence of the `dimensions` arg in `lax.reshape`.